test/client/passwordGenerator.test.js

```diff
@@ -0,0 +1,27 @@
+const { generatePassword } = require('../../client/passwordGenerator');
+
+describe('generatePassword', () => {
+    test('should return a string', async () => {
+        const password = await generatePassword(8);
+        expect(typeof password).toBe('string');
+    });
+
+    test('should return a password with the specified length', async () => {
+        const wordCount = 3
```

Comment:
Do you really need this to be a const in each and every test? Since it's used in the exact same fashion in almost all of the tests, you could pull that out to inside the describe block instead of each individual test block.